### ISSUE TITLE
Control which tab is active when closing a file.

### ISSUE THREAD

**Issue by facelessuser** [facelessuser/TabsExtra#1]
When a tab is closed away form an active tab and the active tab is still open, it should retain the active state.

When the active tab gets closed, maybe it can be configurable as to which tab should be active (left or right of last active tab).

**Comment by PSeitz** [comment]
Is this still working?

I would like to switch to the last active tab when closing a tab, but couldn't find something in th docs

**Comment by facelessuser** [comment]
TabExtra used to have a feature where it would switch tabs when closing a tab. It did alright, but it caused annoying issues when selecting files in the sidebar at times, particularly when you had "preview on click enabled". It isn't easy to filter out such events, so you often get issues where you select a file, and then TabsExtra would select the last active (or left, right, etc.). ST4, which is in a pre-beta state, allows you to select multiple tabs and view them side by side. This adds even more complications.

In the end, I decided to simplify TabsExtra a bit. Tab focusing features really messed with the sidebar which infuriated me when I needed to use the sidebar, so I killed off those features. I'm sorry if they were features that you found useful, but they were already buggy in actuality, and with the latest complications in ST4, I needed to make this plugin easier for me to maintain and less annoying for me to use.

The number one reason I support this plugin and others is that I actually use them. Features that are buggy and cause me pain when I use the plugin, unfortunately, cannot remain.
